README.md

```diff
@@ -133,3 +134,13 @@ SYNC Updating Policy: Test policy 1
 
 The command provides output to identify which actions have been taken. If using a Tyk Gateway, the Gateway will be
 automatically hot-reloaded.
+
+
+## Example: Getting the current Tyk Sync version
```

Comment:
Perhaps change this to:
`Example: Check the currently installed version of Tyk Sync`